src/gt4py/next/ffront/decorator.py

```diff
@@ -495,7 +495,7 @@ def program(*, backend: Optional[ppi.ProgramExecutor]) -> Callable[[types.Functi
 def program(
     definition=None,
     *,
-    backend=None,
+    backend=eve.NOTHING,
```

Comment:
Do you have a concrete comment that you like to have? The docstring of eve.NOTHING explains it already. Should I mention that `None` means _no backend_?

Reply:
Something like this:
```
# If unspecified `DEFAULT_BACKEND`, by default being embedded execution, is used. We use 
# `eve.NOTHING` here instead of `None` since `None` is already used to specify embedded execution.
# As a result we are still able to temporarily switch to a different default backend (e.g. in the 
#  tests to disable execution without explicit backend selection).
```

Reply:
I was just reading this line again yesterday and scratched my head on why exactly we had to use this mechanism and since I was already told I thought a comment would make sense.

Reply:
will add the extended comment in a next PR